File: Python_Ruby/repetition.rb
#While
i = 0
while i < 3 do
    puts('Hello world')
    i = i + 1
end

# 활용

i = 0
while i < 10 do
    puts('puts("Hello world '+(i*9).to_s()+'")')
    i = i + 1
end
    
#조건문 + 반복문

i = 0
while i < 10 do
  if i == 4
    puts(i)
  end
  i = i + 1
end

###################    
i = 0
while i < 10 do
    if i == 4
        break
    end
    puts(i)
    i = i + 1
end
print('after while')

File: Python_Ruby/repetition.py
#While

while False:
    print('Hello world')
print('After while')

#반복 조건

i = 0
while i < 3:
    print('Hello world')
    i = i + 1


# 활용

i = 0
while i < 10:
    print('print("Hello world '+str(i*9)+'")')
    i = i + 1
    
#조건문 + 반복문

i = 0
while i < 10:
    if i == 4:
        print(i)
    i = i + 1

###################    
i = 0
while i < 10:
    if i == 4:
        break
    print(i)
    i = i + 1
print('after while')